src/widgets/editbox.rs

```diff
@@ -314,7 +343,7 @@ impl<'a> Editbox<'a> {
         let pos = context
             .window
             .cursor
-            .fit(size, Layout::Free(Vector2::new(5., 5.)));
+            .fit(size, Layout::Free((5., 5.).into()));
```

Comment:
I like Vector2::new() a bit more than .into() :) 

And I think that this is outside of the scope of this PR anyway.

Reply:
I revert this changes